template/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--CDN https://handlebarsjs.com/installation/#cdns -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.7.7/handlebars.js"></script>
+  </head>
+  <body>
+    <div id="contentDiv"></div>
+
+    <script id="entry-template" type="text/x-handlebars-template">
+      <div class="entry">
+        {{#if knowledge_panels}}
+          <div class="panel item">
+            {{#each knowledge_panels}}
+              {{#each this}}
+                {{#if elements.length}}
+                  <h3 class="panel title">{{title_element.title}}</h3>
+                  {{#each elements}}
+                    {{{text_element.html}}}
+                    {{#if text_element.source_url}}
+                      <em>source:<a
+                          href="{{text_element.source_url}}"
+                        >{{text_element.source_text}}</a></em>
+                    {{/if}}
+                  {{/each}}
+                {{/if}}
+              {{/each}}
+            {{/each}}
+          </div>
+        {{/if}}
+      </div>
+    </script>
+
+    <script type="text/javascript">
+      // Assign the template name
+      var source = document.getElementById("entry-template").innerHTML;
+      var template = Handlebars.compile(source);
+
+      // Compile the template
+      var context = {
+        // Response from the API
+      };
```

Comment:
@alexgarel question: How I can inject the API response I.e, "/knowledge_panel?{query}" into the "context"